src/renderer/hooks/useBroadcast.js

```diff
@@ -28,12 +29,17 @@ export const useBroadcast = ({ account, parentAccount }: SignTransactionArgs) =>
         return Promise.resolve(signedOperation.operation);
       }
 
-      return execAndWaitAtLeast(3000, () =>
-        bridge.broadcast({
+      return execAndWaitAtLeast(3000, async () => {
+        const operation = await bridge.broadcast({
           account: mainAccount,
           signedOperation,
-        }),
-      );
+        });
+        log(
+          "transaction-summary",
+          `✔️ broadcasted! optimistic operation: ${formatOperation(mainAccount)(operation)}`,
+        );
+        return operation;
```

Comment:
I think it's already covered inside `bridge.broadcast` because it in fact use the "proxy commands" implicitly

could you check @machard ?

i suspect we have a double log happening.